python/libraries/SystemIO.py

```diff
@@ -8,6 +10,15 @@ async def inp(question):
     with ThreadPoolExecutor(1, "AsyncInput") as executor:
         return await asyncio.get_event_loop().run_in_executor(executor, input, question)
 
+def run(command):
+    can_run = os.environ.get("COMMAND_ALLOW") == "true"
+    if not can_run:
+        return 1
+
+    return os.system(COMMAND_ALLOW)
```

Comment:
What if this throws an error?

Reply:
It returns a `0` if it works or a `1` if it does not, I don't think it can throw an error.

Reply:
Oh, oops, was looking at the wrong type definition.